Session-3/CurrencyExchangeApp/CurrencyExchangeApp/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CurrencyConverter;
+using RestSharp;
+using System.Runtime.Serialization;
+
+namespace CurrencyExchangeApp
+{
+    internal class Program
+    {
+        static int Main(string[] args)
+        {
+            string to ="" ;
```

Comment:
Format code properly. Also why do we need int return type for Main? Change it to void